cluster/gce/gci/configure-helper.sh

```diff
@@ -3060,6 +3071,13 @@ EOF
       cni_template_path=""
     fi
   fi
+
+   # Use systemd cgroup driver when running on cgroupv2
+  local systemdCgroup="false"
+  if [[ "${CGROUP_CONFIG-}" == "cgroup2fs" ]]; then
```

Comment:
Might be good to add a default value in config-default.sh

Reply:
I thought about it, but I don't think it makes sense in this context.

The issue is that we want to default to the systemd cgroup driver when running on cgroupv2 OS image. `config-default.sh` sets defaults ahead of time for deployment. At that time, we don't know ahead if the OS image will be cgroupv2 enabled or not.

As a result, this check is done when the VM actually boots which is when we can detect the cgroup configuration. Let me know what you think and if that makes sense.